### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by system1system2** [0xALIVEBEEF/Expanded-XY-grid#4]
I published this idea on the AUTOMATIC1111 Discussion forum [here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/5123), but now I realize that maybe Expanded-XY-grid already does it:

> Right now, the X/Y Plot script for AUTOMATIC1111 can already compare multiple models with the Checkpoint name type.
> It works well if you are comparing models that have no activation tokens (e.g.: SD 1.4 vs 1.5 vs 2.0).
> 
> But it doesn't work well if, for example, you want to compare classicAnim-v1.ckpt vs moDi-v1-pruned.ckpt vs robo-diffusion-v1.ckpt, etc. And that's because these checkpoint need an activation token: classic disney style, modern disney style, nousr robot, etc.
> 
> You could theoretically use Checkpoint name for the X type and Prompt S/R for the Y type, but it would produce a lot of unnecessary outputs.
> 
> It would be much better if the X/Y Plot script could associate an activation token to each checkpoint in the X type.

Is it possible to do it with your script?

**Comment by 0xALIVEBEEF** [comment]
No, my script does not do that currently. I could update it, but it seems like that would require somewhat significant changes to the code and testing to see if it works. You can do Prompt S/R and Prompt S/R placeholder on the prompt. That could work.

Like:
Prompt: style, darth vader riding a bicycle
X params: Multitool: Checkpoint names: ... | Prompt S/R: style, classic disney style, modern disney style, nousr robot

But I've never used activation tokens, so I don't know how they work
